### ISSUE TITLE
ui5-link does not support aria-label

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#2356]
**Describe the bug**
The ui5-link component does not pass any aria attributes down to the anchor tag inside it, so screenreaders only read the inner text of the link.

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Link/ in chrome
2. using dev tools, add an aria-label to a ui5-link
3. notice the anchor tag in the shadow root does not get the aria-label
4. using any screenreader, focus on the link and notice the aria-label is not read out.

**Expected behavior**
The aria-label and aria-labelledby attributes should be respected by ui5-link and passed to the inner anchor element to allow compliance with accessibility standards.

**Screenshots**
![Screen Shot 2020-10-14 at 8 40 37 AM](https://user-images.githubusercontent.com/24360164/96016393-88539c80-0dfd-11eb-949d-f4bafca5f319.png)

**Context**
- UI5 Web Components version: any
- screen reader: NVDA, JAWS, Mac VoiceOver

**Affected components** 
ui5-link

**Organization:**
SuccessFactors

**Priority:** 
Medium

Thank you!
